code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -689,7 +689,8 @@ Code:
 				dat += "<center>Please select alert type:<br>"
 				dat += "<a href='?src=\ref[src];alert=1'>Medical Alert</a><br>"
 				dat += "<a href='?src=\ref[src];alert=2'>Engineering Alert</a><br>"
-				dat += "<a href='?src=\ref[src];alert=3'>Security Alert</a>"
+				dat += "<a href='?src=\ref[src];alert=3'>Security Alert</a><br>"
+				dat += "<a href='?src=\ref[src];alert=4'>Janitor Alert</a>"
```

Comment:
This whole block would be more performant as a single string: (Byond's not great at string building)
```
				dat += "{
<center>Please select alert type:<br>
<a href='?src=\ref[src];alert=1'>Medical Alert</a><br>"
<a href='?src=\ref[src];alert=2'>Engineering Alert</a><br>"
<a href='?src=\ref[src];alert=3'>Security Alert</a><br>"
<a href='?src=\ref[src];alert=4'>Janitor Alert</a>
}"
```
(I can't do this as a code suggestion unfort')

Reply:
Too many "!
Implemented.